docs/common-issues.md

```diff
@@ -92,7 +103,7 @@ in the review queue.`
 
 ## Zulip
 
-### Student pings mentors constantly for reviews
+### Student pings mentors for reviews instead of going through the queue system
```

Comment:
I think the old title is better; it's OK if students do this from time-to-time (sometimes we have lost their task or just did the small change we asked them to do), so it's really only a problem for those students who are doing this on a daily basis (or even more often!).